controllers/fluentbitconfig_controller.go

```diff
@@ -102,12 +103,13 @@ func (r *FluentBitConfigReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		if err != nil {
 			return ctrl.Result{}, err
 		}
-		if err = r.List(ctx, &parsers, client.InNamespace(req.Namespace), client.MatchingLabelsSelector{Selector: selector}); err != nil {
+		if err = r.List(ctx, &parsers, client.MatchingLabelsSelector{Selector: selector}); err != nil {
 			return ctrl.Result{}, err
 		}
 
+		ns := os.Getenv("MY_POD_NAMESPACE")
```

Comment:
MY_POD_NAMESPACE => NAMESPACE will make sense? And k8s doesn't use NAMESPACE as an env value.